tst/performance/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+
+##========================================================================================
+## Athena++ astrophysical MHD code
+## Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+## Licensed under the 3-clause BSD License, see LICENSE file for details
+##========================================================================================
+## (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+##
+## This program was produced under U.S. Government contract 89233218CNA000001 for Los
+## Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+## for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+## in the program are reserved by Triad National Security, LLC, and the U.S. Department
+## of Energy/National Nuclear Security Administration. The Government is granted for
+## itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+## license in this material to reproduce, prepare derivative works, distribute copies to
+## the public, perform publicly and display publicly, and to permit others to do so.
+##========================================================================================
+
+list(APPEND performance_tests_SOURCES
```

Comment:
If nobody's going to iterate over this variable, I would just change this to

```cmake
add_executable(performance_tests)
target_sources(
    performance_tests
    PRIVATE
        test_container_iterator.cpp
)
```

Reply:
Done.